packages/zapp/console/src/components/Executions/ExecutionDetails/Timeline/ExecutionTimeline.tsx

```diff
@@ -84,29 +79,12 @@ export const ExecutionTimeline: React.FC<ExProps> = ({ chartTimezone }) => {
   const [originalNodes, setOriginalNodes] = useState<dNode[]>([]);
   const [showNodes, setShowNodes] = useState<dNode[]>([]);
   const [startedAt, setStartedAt] = useState<Date>(new Date());
-
-  const { compiledWorkflowClosure } = useNodeExecutionContext();
-  const { chartInterval: chartTimeInterval } = useScaleContext();
-  const { staticExecutionIdsMap } = compiledWorkflowClosure
-    ? transformerWorkflowToDag(compiledWorkflowClosure)
-    : [];
-
   const nodeExecutionsById = useContext(NodeExecutionsByIdContext);
-
-  const dynamicParents = checkForDynamicExecutions(nodeExecutionsById, staticExecutionIdsMap);
-
-  const { data: dynamicWorkflows } = useQuery(
-    makeNodeExecutionDynamicWorkflowQuery(dynamicParents),
-  );
+  const { chartInterval: chartTimeInterval } = useScaleContext();
 
   useEffect(() => {
-    const nodes: dNode[] = compiledWorkflowClosure
-      ? transformerWorkflowToDag(compiledWorkflowClosure, dynamicWorkflows).dag.nodes
-      : [];
-    // we remove start/end node info in the root dNode list during first assignment
-    const initializeNodes = convertToPlainNodes(nodes);
     setOriginalNodes(initializeNodes);
-  }, [dynamicWorkflows, compiledWorkflowClosure]);
+  }, [initializeNodes]);
```

Comment:
this effect is useless, set originalNodes directly upon var initialization
line 79 should look like this `const [originalNodes, setOriginalNodes] = useState<dNode[]>(initializeNodes);` - if initializedNodes changes, this whole component re-renders

Reply:
done